meta/terminal42/contao-url-rewrite/de.yml

```diff
@@ -0,0 +1,11 @@
+de:
+  title: Folder page
```

Comment:
`Folder page`? :)

Reply:
Not of course not. Was too much into making sure that I did not have typos that I did not look at the copy/paste leftovers